analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -257,10 +266,66 @@ public void perform() {
                     alignmentModel.setGroup(commonPrefix);
                 }
 
+                logger.info("Completed processing for alignment and writing {} ", cache);
+                logger.info("Creating PME alignment JSON file");
+                final PME jsonReport = new PME();
+                final List<ModulesItem> modules = jsonReport.getModules();
+                final String originalVer = alignmentModel.getOriginalVersion();
+                final ModulesItem rootModule = new ModulesItem();
+                final String originalGav = alignmentModel.getGroup() + ":" + alignmentModel.getName() + ":"
+                        + alignmentModel.getOriginalVersion();
+                rootModule.getGav().setOriginalGAV(originalGav);
+                logger.debug("Original GAV set to {}", rootModule.getGav().getOriginalGAV());
+                final String gav = alignmentModel.getGroup() + ":" + alignmentModel.getName() + ":"
+                        + alignmentModel.getVersion();
+                final ProjectVersionRef pvr = SimpleProjectVersionRef.parse(gav);
+                rootModule.getGav().setPVR(pvr);
+                logger.debug("PVR set to {}", rootModule.getGav().getPVR());
+                modules.add(rootModule);
+                append(builder, "------------------- project {}", originalGav);
+                final String newVer = alignmentModel.getVersion();
+
+                if (!originalVer.equals(newVer)) {
+                    logger.debug("Project version : {} ---> {}", originalVer, newVer);
+                    append(builder, "\tProject version : {} ---> {}", originalVer, newVer);
+                }
+
+                projectDependencies.forEach((key, value) -> {
+                    if (configuration.versionModificationEnabled()) {
+                        final ManipulationModel correspondingModule = alignmentModel.findCorrespondingChild(key);
+                        final String group = correspondingModule.getGroup().isEmpty() ? alignmentModel.getGroup()
+                                : correspondingModule.getGroup();
+                        final String artifact = correspondingModule.getName();
+                        final String ga = group + ":" + artifact;
+                        final String originalModuleGav = ga + ":" + (correspondingModule.getOriginalVersion() == null
+                                ? ""
+                                : correspondingModule.getOriginalVersion());
+                        final String newModuleGav = ga + ":" + correspondingModule.getVersion();
+                        logger.debug("Relocation : {} ---> {}", originalModuleGav, newModuleGav);
+                        append(builder, "\tRelocation : {} ---> {}", originalModuleGav, newModuleGav);
```

Comment:
I am seeing project sub-modules being labelled as relocations which does not seem right.

Reply:
OK, then this isn't entirely correct. Is it only printing submodules, or submodules are included and shouldn't be?

I guess I had trouble understanding the distinction as all project "dependencies" were also called "modules" or "children".

Reply:
It would help if there were tests (and eventually documentation) I think. Regardless, a submodule version change is not a relocation I would think

Reply:
Yes, but it's not really possible to write a test of the output files if the output cannot be confined to a directory specific to the test being run. It's possible to have a unit test with a pre-existing test resource, I suppose, but what about the functional tests?

If there are tests already doing this, I am not sure, it looks like all of the files are confined to the project root, but this is a problem, don't we want to copy all the test resources over before running the tests anyway?